docs/guides/modules/test/pages/smarter-testing.adoc

```diff
@@ -41,565 +41,1571 @@ TIP: In codebases with sparse test coverage, Smarter Testing cannot accurately d
-version: 2.1
-jobs:
-  test:
-    executor: node-with-service
-    steps:
-      - setup
-      - run: JEST_JUNIT_OUTPUT_FILE="test-reports/tests.xml" jest --runInBand --reporters=jest-junit --bail --testPathPattern=src/
-      - store_test_results:
-          path: test-reports
+$ sha256sum -c <(echo "04f37e47a12b4677cda35b804357b10c5891ef42081f1674fe74d82f56baf3cb  ./circleci-testsuite.gz")
 ----
 
-=== 1.3 Populate the discover command
-
-The `discover` command finds all tests for a given test suite. The stdout of this command should be a space-separated or newline-separated list of tests that will be passed into the run command.
+Mac Intel::
+https://circleci-binary-releases.s3.amazonaws.com/circleci-cli-plugins/circleci-testsuite/1.0.15108-d839eb2/darwin/amd64/circleci-testsuite.gz[Download]
++
+Verify with:
++
+[source,console]
+----
+$ sha256sum -c <(echo "8c933b3358d8659ea7cc228b2fbd83b4e605c472c0b60355b3b0f97d0952852b  ./circleci-testsuite.gz")
+----
 
-The `discover` command should not execute the tests.
+Linux AMD64::
+https://circleci-binary-releases.s3.amazonaws.com/circleci-cli-plugins/circleci-testsuite/1.0.15108-d839eb2/linux/amd64/circleci-testsuite.gz[Download]
++
+Verify with:
++
+[source,console]
+----
+$ sha256sum -c <(echo "f189936b32bae5cde22a7522dc58bd4e988c1384f690801c4cc88dfebf90319c  ./circleci-testsuite.gz")
+----
 
-All output will be used as tests, any tool that prints metadata in the stdout will need to be omitted.
+Linux ARM::
+https://circleci-binary-releases.s3.amazonaws.com/circleci-cli-plugins/circleci-testsuite/1.0.15108-d839eb2/linux/arm/circleci-testsuite.gz[Download]
++
+Verify with:
++
+[source,console]
+----
+$ sha256sum -c <(echo "b84ccd76ddb4bad1e8226381e308f9a9d95e2314cd3676fcdaffcf234446fcd5  ./circleci-testsuite.gz")
+----
 
-. Update the `.circleci/test-suites.yml` file with the discover command.
+Linux ARM64::
+https://circleci-binary-releases.s3.amazonaws.com/circleci-cli-plugins/circleci-testsuite/1.0.15108-d839eb2/linux/arm64/circleci-testsuite.gz[Download]
++
+Verify with:
++
+[source,console]
+----
+$ sha256sum -c <(echo "a5770f0c964b434df3560358836fa2401f526379e84aac2cd6c7946180ed1ce6  ./circleci-testsuite.gz")
+----
 
-[source,yaml]
+Windows AMD64::
+https://circleci-binary-releases.s3.amazonaws.com/circleci-cli-plugins/circleci-testsuite/1.0.15108-d839eb2/windows/amd64/circleci-testsuite.exe.gz[Download]
++
+Verify with:
++
+[source,console]
 ----
-# .circleci/test-suites.yml
-name: ci tests
-discover: jest --listTests --testPathPattern=src/
-run: ##
-outputs:
-  junit: ##
+$ sha256sum -c <(echo "ca0f395b1ffeca557428a30cc405ad193fe681792822cfae73c7c7c813687c0c  ./circleci-testsuite.exe.gz")
 ----
+====
 
-*Examples for the `discover` command*
+After downloading the plugin and verifying the checksum:
 
-[.table-scroll]
---
-[cols=2*, options="header"]
-|===
-| Test Framework| Command
+. Decompress the binary
+. Set the execute bit
+. Move it to a directory on your shell's `$PATH`
```

Comment:
```suggestion
. Decompress the binary.
. Set the execute bit.
. Move it to a directory on your shell's `$PATH`.
```

Reply:
Done (I think? Probably got the source-in-list formatting wrong)